CMakeLists.txt

```diff
@@ -81,6 +81,13 @@ target_link_libraries(demo ${catkin_LIBRARIES})
 # python bindings #
 ###################
 
+# generic #
+pybind11_add_module(py_generic srcpy/py_generic.cpp)
```

Comment:
am I correct this will result in a python package called py_generic ? (as in : import py_generic). If so, are more specific name could be better.

Reply:
It will be `robot_interfaces.py_generic`, so not polluting the global namespace. There is also the `import *` in the `__init__.py`, so the user would never need to actually use that name (`from robot_interfaces import Status` would work).